massdriver.yaml

```diff
@@ -12,21 +12,27 @@ params:
     - __name: Large Network (65K IPs)
       cidr: "10.0.0.0/16"
   required:
-    - azure_region
-    - cidr
+    - network
   properties:
-    azure_region:
-      description: Select the Azure region you'd like to provision your resources in. This cannot be changed after the resource is created.
-      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/azure-region.json
-      $md.immutable: true
-    cidr:
-      title: "Virtual Network CIDR"
-      type: string
-      description: Enter a CIDR IP address range to use for the size of your virtual network. This cannot be changed after the resource is created.
-      $md.immutable: true
-      pattern: ^(?:10\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])|172\.(?:1[6-9]|2[0-9]|3[0-1])|192\.168)(?:\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){2}(?:\/(1[6-9]|20))?$
-      message:
-        pattern: "Range must be from private networking space (10.X.X.X, 172.16-31.X.X, 192.168.X.X). Mask must be between 8 and 24"
+    network:
```

Comment:
I think since we removed `additionalProperties: false`, this _can_ work. A provisioned bundle would have `cidr` set and the validation wouldn't be happy with a bundle published after that w/o `cidr`.

Also, we should test this but I think it's possible to get things in a bad state.

+ old bundle is provisioned, `cidr` is 10.0.0.0/16
+ new bundle is published, requires `network.cidr` to be set and since it hasn't yet, it's a blank field
+ user enters `10.1.0.0/16` which is different and triggers a destroy / recreate
+ opa catches it, bundle is borked, `network.cidr` is immutable in the FE

Reply:
Yep, I set up this PR in hopes that no one is using Azure infra outside of sandbox yet. Won't merge w/o confirmation.